src/layout/IHyprLayout.hpp

```diff
@@ -133,6 +133,10 @@ interface IHyprLayout {
     */
     virtual void         onWindowFocusChange(CWindow*);
 
+    /*
+      Add virtual descontruct fiunc
+    */
+    virtual ~IHyprLayout()=0;
```

Comment:
destructors should be right below the constructor, or on the top if one is not present. Furthermore, useless comment, and styling: no spaces around the `=`